run-queue

```diff
@@ -64,12 +64,21 @@ def consume_webhook_queue(channel, amqp):
         elif action == 'closed' and pull_request.get('merged', False):
             sha = pull_request['head']['sha']
             db = os.path.join(get_images_data_dir(), "test-failures.db")
+            data = os.path.join(get_images_data_dir(), "failure-stats.js")
             cmd = ['./store-failures', "--db", db, "--repo", repo, sha]
             body = {
                 "command": " ".join(cmd),
             }
             channel.basic_publish('', 'statistics', json.dumps(body),
                                   properties=pika.BasicProperties(priority=distributed_queue.MAX_PRIORITY))
+
+            cmd = ['./generate-stats', "--db", db, ">", data]
```

Comment:
This is "racy", but I think it is fine, we don't necessary need to have new with the very last failures.
Also not sure about the `>` part...

Reply:
It looks very confusing as an argv list, one would rather expect to use `stdout=open(data)` somewhere. But this is an actual shell string. Please skip building a list and plug in the string straight into `body = { "command": "./generate..." }` so that it's immediately clear that this is *not* an argv. The same is true for the existing ./store-failures call.

As for the race, that's indeed nasty as this may be done by a different bot. Why not simply do it in one command?

    ./store-failures ... && ./generate-stats

Reply:
Right, Having just one string with `&&` seems like the best option. Updated to that.